### ISSUE TITLE
Some clarifications and README.md feedback

### ISSUE THREAD

**Issue by anthonyra** [wealdtech/execd#3]
I'm working on a project that I intend to have an archival node running Erigon as the execution client. Since the Merge on Ethereum the execution client and consensus client are separated. In the README it talks about Erigon as a consensus node. I noticed your commit a few hours ago so I'm assuming it works post Merge still?

Are there any limitations to what `execd` migrates to PostgreSQL? Just curious how it compares to something like `ethereum-etl` for schema and tables.

I noticed while looking at the README was that the "notes on the tables" results in a 404 now. 

TL;DR - Does `execd` still works as an ETL using Erigon to PostgreSQL? If it still does, I'd be willing to contribute to align this as necessary to any standards or norms (if there's any deviation) along with ensuring the README is updated from my experience of integrating this in my pipeline.

**Comment by mcdee** [comment]
The docs were incorrect, erigon is an execution node.  You will need a connected consensus node to sync the execution node, however.

The  schema can be found at https://github.com/wealdtech/execd/blob/master/services/execdb/postgresql/upgrader.go

This project is active and updated in line with changes in the Ethereum block and transaction space.

**Comment by anthonyra** [comment]
Awesome! Thank you for clarifying and providing me the schema source.
